lib/kvdb/ikvdb.c

```diff
@@ -1770,11 +1762,11 @@ ikvdb_config_attach(struct ikvdb *kvdb, struct config *conf)
 }
 
 bool
-ikvdb_read_only(struct ikvdb *handle)
+ikvdb_allows_user_writes(struct ikvdb *ikvdb)
```

Comment:
Can this be const?

Reply:
Tried when I added this function but ran into some errors...